<launch>
	<group ns="ekf_general">
	    <node pkg="ekf_general" name="localization_node" type="localization_node" output="screen">
	        <param name="path_2_map" value="$(find ekf_general)/DataSets/map_o3.txt" />
	        <param name="sens_top_name" value="sensors_sim" />
	        <param name="map_srv_name" value="ekf_output_srv" />
	        <param name="nu_est_top" value="nu_estimated" />
	        <param name="nu_real_top" value="nu_real" />
	        <param name="error_top" value="angle_error" />
	        <param name="initial_R" value="0.0001" />
	        <param name="initial_Q" value="0.0001" />
	        <param name="delta" value="0.999" />
	        
		</node>

		<node pkg="ekf_general" name="sens_sim_node" type="sens_sim_node" output="screen">
			<param name="path_2_sens" value="$(find ekf_general)/DataSets/so_o3_ie.txt"/>
			<param name="sensors_top" value="sensors_sim"/>
		</node>

		<node pkg="ekf_general" name="output_node" type="output_node.py" output="screen">
			<param name="top_output" value="ekf_output_srv"/>
			<param name="top_mu_est" value="nu_estimated"/>
			<param name="top_mu_real" value="nu_real"/>
		</node>
	</group>
</launch>